cooltools/cli/compute_expected.py

```diff
@@ -47,41 +47,35 @@
     "of a Hi-C map.",
     type=click.Choice(['cis', 'trans']),
     default='cis',
-    show_default=True,
-    )
+    show_default=True)
+@click.option(
+    "--balanced/--no-balance",
```

Comment:
```suggestion
    "--balance/--no-balance",
```

Reply:
more consistencies to be fixed later between `cooler` and `cooltools`, e.g. https://github.com/mirnylab/cooler/blob/843dadca5ef58e3b794dbaf23430082c9a634532/cooler/cli/dump.py#L273